README.md

```diff
@@ -770,6 +768,15 @@ Table of Contents
   * [blockspring.com](https://www.blockspring.com/) — Cloud functions. Free for 5 million runs/month
   * [BMC Developer Program](https://developers.bmc.com/site/global/bmc_helix_platform/program/overview/index.gsp) — The BMC Developer Program provides documentation and resources to build and deploy digital innovations for your enterprise. Access to a comprehensive, personal sandbox which includes the platform, SDK, and a library of components that can be used to build and tailor apps.
   * [darklang.com](https://darklang.com/) - Hosted language combined with editor and infrastructure. Free during the beta, generous free tier planned after beta.
+  * [Firebase](https://firebase.com) — Firebase helps you build and run successful apps. Offers a variety of services in the Free Spark Plan listed here :
+    - Authentication - 10K Phone Auths/month (seperate 10K limit for US,Canada & India) & unlimited auths for other methods.
+    - Cloud Firestore,Cloud Functions & BigQuery - same as offered in GCP Always Free Tier.
+    - Hosting - 10GB storage with 360 MB data transfer per day with custom domain & SSL.
+    - Firebase ML - Custom Model Deployment.
+    - Realtime Database - 100 simultaneous connections with 1GB storage & 10GB downloads per month.
+    - Cloud Storage - 5GB storage with 1GB/day upto 50k/day downloads and 20k/day upload operations.
+    - Testlab - 10 Virtual Device & 5 Physical Device tests per day.
+    - Always Free :A/B Testing, Analytics, App Distribution, App Indexing,Cloud Messaging (FCM), Crashlytics, Dynamic Links, In-App Messaging, Performance Monitoring, Predictions, and Remote Config.
```

Comment:
isnt this technically part of google cloud? I think it should be a condense one line by the existing big google cloud entry.

Reply:
The Cloud Firestore,Cloud Functions & Big Query are a part of GCP.
But the other services are exclusively a part of Firebase (although they're hosted on GCP)